lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -366,6 +366,7 @@ dummyTransactionLayer = TransactionLayer
         error "dummyTransactionLayer: estimateSize not implemented"
     , estimateMaxNumberOfInputs =
         error "dummyTransactionLayer: estimateMaxNumberOfInputs not implemented"
+    , validateSelection = \_ -> pure ()
```

Comment:
It'd be better to actually throw an `error not implemented` here too. Providing a (wrong) default implementation may lead to unpleasant surprise if we suddenly start using this branch. If we haven't implemented this, let's say it clear and loud :)

Reply:
previously as validateSelection was used in `_signTx` the above hack was needed. now as we moved everything to `mkStdTx` we can provide `error not implemented`